.github/workflows/CI.yml

```diff
@@ -223,6 +223,26 @@ jobs:
           command: fmt
           args: --all -- --check
 
+  clippy:
+    name: clippy
+    runs-on: ubuntu-latest
+    container:
+      image: ubuntu:20.10
+    steps:
+      - run: apt-get update -y
+      - run: apt-get install -y libgtk-3-dev libglib2.0-dev libgraphene-1.0-dev git xvfb curl libcairo-gobject2 libcairo2-dev
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
As mentioned a few times on IRC yesterday, running nightly clippy is not a good idea. It contains unstabilized lints and is regularly changing, which is suboptimal for a CI. We don't want random contributor's PRs to fail just because last night a new lint was added and some completely unrelated code triggers it.

Reply:
I'm not sure whether we can get back to using stable for clippy, unless we removed the --all-features and we go with manually speechifying the highest feature ?

Reply:
I think that's the only option, yes

Reply:
This one is the only one left from what I can see, then we can merge it

Reply:
I think it should be ready, let's see how the CI goes